batch_convert.qml

```diff
@@ -41,14 +42,16 @@ MuseScore {
         }
         else
             batchConvert.visible = true // needed for unknown reasons
+
+        if(settings.ePath===undefined) settings.ePath=settings.iPath;
     }
 
     //Window {
     id: batchConvert
 
     // `width` and `height` allegedly are not valid property names, works regardless and seems needed?!
-    width: mainRow.childrenRect.width
-    height: mainRow.childrenRect.height
+    width: mainRow.childrenRect.width + 40
```

Comment:
This value seems to depend on the translation of "Use Regular Expressions" to not be (much) longer than the original, doesn't it?

Reply:
No. This is to add some margins around the mainRow.childrenRect. This is the negative side of setting the plugin's dimension to the top element's childrenRect size. There is no room around the content, no margin. And I haven't found a way to add margin/padding to the plugin's top element, or the mainRow that would be taken into account in the childrenRect dimension.

Whatever the length of the  "Use Regular Expression" text, it will fit nicely.

![image](https://user-images.githubusercontent.com/50494/187639977-ee42b60a-2151-4143-b780-55b769ccd24a.png)

Reply:
Ok, I see, it is basically 2*20, as set further down too